inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -311,6 +316,14 @@ void LayerTestsCommon::ConfigureNetwork() {
             out.second->setPrecision(outPrc);
         }
     }
+
+    std::map<std::string, ngraph::PartialShape> shapes;
+    auto isdm = cnnNetwork.getInputsInfo();
+    for (auto&& idm : isdm) {
+        shapes[idm.first] = isdm[idm.first]->getInputData()->getPartialShape();
```

Comment:
looks like this line redundant

Reply:
Isn't it necessary for copying from `PartialShape` to `Shape`? I think I didn't understand your point, and could you explain more about it? And I changed the names of full name.

Reply:
i mean that you assign to `shapes[idm.first] = isdm[idm.first]->getInputData()->getPartialShape()` and after assign `std::vector<ngraph::Dimension>(inputDynamicShape)`
so looks like first assignment redundant

Reply:
I thought the casting was effective for Dimension only. And I fixed it as you commented, thanks.